src/components/modals/options/AnimationOptionsModal.vue

```diff
@@ -76,6 +82,8 @@ export default {
       this.tachyonsUnlocked = this.realityUnlocked || Currency.tachyonParticles.gt(0);
       this.animatedThemeUnlocked = Theme.animatedThemeUnlocked;
       this.isS11Active = Theme.currentName() === "S11";
+      this.isS11Unlocked = Themes.available().map(t => t.name).includes("S11");
+      this.isBlackHoleUnlocked = player.blackHole[0].unlocked;
```

Comment:
Should probably change this to `BlackHoles.areUnlocked` for consistency with other checks elsewhere in the code